internal/cluster/distribution/eks/eksprovider/driver/cluster_creator.go

```diff
@@ -467,6 +469,24 @@ func (c *EksClusterCreator) validate(r *pkgCluster.CreateClusterRequest, logger
 		}
 	}
 
+	tagValidationErrs := make([]error, 0)
```

Comment:
```suggestion
	tagValidationErrs := make([]error, 0, len(r.Properties.CreateClusterEKS.Tags))
```

Preallocating makes sense even if you only know an upper bound, because you can spare reslicing.